src/CowSdk.ts

```diff
@@ -6,29 +6,37 @@ import { SupportedChainId as ChainId, SupportedChainId } from './constants/chain
 import { Context, CowContext } from './utils/context'
 import { signOrder, signOrderCancellation, UnsignedOrder } from './utils/sign'
 import { ZeroXApi } from './api/0x'
-import { MatchaOptions } from './api/0x/types'
+import { ZeroXOptions } from './api/0x/types'
 import ParaswapApi from './api/paraswap'
+import { ParaswapOptions } from './api/paraswap/types'
+import { WithEnabled } from './types'
 
 type Options = {
   loglevel?: LogLevelDesc
-  matchaOptions?: MatchaOptions
+  zeroXOptions?: Partial<ZeroXOptions & WithEnabled>
+  paraswapOptions?: Partial<ParaswapOptions> & WithEnabled
```

Comment:
here the intersection of `WithEnabled` is done inside the Partial in `zeroXOptions` and outside in `paraswapOptions`. Both work the same as `enabled` is optional by default but it makes you think.

Reply:
hahah shit. ok good point let me fix this. good 👁️

Reply:
done in #60